roofit/roofitcore/src/RooMinimizer.cxx

```diff
@@ -959,5 +959,9 @@ RooFitResult* RooMinimizer::lastMinuitFit(const RooArgList& varList)
   return res;
 
 }
+Int_t RooMinimizer::getPrintLevel() const
```

Comment:
I would not outline these one-liners. They should better be inlined in the header file. There are also many such one-liners in RooAbsMininmizerFcn, but since they are copy pasted from the original file, I'm fine with keeping them there. But for new code, it would be nice to inline.